web/profiles/custom/os2loop/modules/os2loop_upvote/src/Helper/FlagHelper.php

```diff
@@ -3,25 +3,41 @@
 namespace Drupal\os2loop_upvote\Helper;
 
 use Drupal\Core\Cache\Cache;
+use Drupal\flag\FlagCountManager;
 
 /**
  * Flaghelper for creating top comment and count flags.
  */
 class FlagHelper {
+  /**
+   * The flag service.
+   *
+   * @var \Drupal\flag\FlagCountManager
+   */
+  protected $flagService;
+
+  /**
+   * Constructor.
+   *
+   * @param \Drupal\flag\FlagCountManager $flag_service
+   *   The flag service.
+   */
+  public function __construct(FlagCountManager $flag_service) {
```

Comment:
`$flag_service` (and `$this->flagService`) should be renamed to `$flagCounterManager` because that's what it is.